base/ubi9-python-3.11/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM registry.access.redhat.com/ubi9/python-311:latest
+
+LABEL name="odh-notebook-base-ubi9-python-3.9" \
+      summary="Python 3.9 base image for ODH notebooks" \
+      description="Base Python 3.9 builder image based on UBI9 for ODH notebooks" \
+      io.k8s.display-name="Python 3.9 base image for ODH notebooks" \
+      io.k8s.description="Base Python 3.9 builder image based on UBI9 for ODH notebooks" \
+      authoritative-source-url="https://github.com/opendatahub-io/notebooks" \
+      io.openshift.build.commit.ref="main" \
+      io.openshift.build.source-location="https://github.com/opendatahub-io/notebooks/tree/main/base/ubi9-python-3.9" \
+      io.openshift.build.image="quay.io/opendatahub/workbench-images:base-ubi9-python-3.9"
```

Comment:
I build a test build of some of the images and noticed that we may want to update this to reflect the _python-3.11_ instead of _python-3.9_ in the labels here. My apologies if you are already aware.

Cheers,
Tipton

Reply:
Thanks! Good catch! Fixed in 7ba66c703d293c4b57572b8621c0d1ac1f512f80